packages/components/src/components/quickfact/quickfact.ts

```diff
@@ -0,0 +1,143 @@
+import '../icon/icon';
+import { css, html } from 'lit';
+import { customElement } from '../../internal/register-custom-element';
+import { property } from 'lit/decorators.js';
+import { setDefaultAnimation } from '../../utilities/animation-registry';
+import cx from 'classix';
+import SdAccordion from '../accordion/accordion';
+
+/**
+ * @summary Accordion shows a brief summary and expands to show additional content.
+ * @documentation https://solid.union-investment.com/[storybook-link]/quickfact
+ * @status stable
+ * @since 3.12.0
+ *
+ * @slot icon - Optional icon to show in the header. Works best with `<sd-icon>`.
+ *
+ * @csspart icon - The container that wraps the icon.
+ *
+ * @dependency sd-icon sd-accordion
+ */
+@customElement('sd-quickfact')
+export default class SdQuickfact extends SdAccordion {
+  /**
+   * Determines if the quickfact is not interactive. When set to `true`, the quickfact will not expand or collapse.
+   */
+  @property({ type: Boolean, reflect: true }) expandable = false;
```

Comment:
![CleanShot 2024-08-02 at 16 08 26@2x](https://github.com/user-attachments/assets/36b3176a-a6c8-4554-ac2c-a2c1f04c913d)

Shouldn't have a tabindex in that case.